Scripts/Python/clftImager.py

```diff
@@ -433,15 +433,15 @@ def OnNotify(self,state,id,events):
             cam.enableFirstPersonOverride()
             #PtEnableForwardMovement()
 
-            if (self.EndgameSolved()):
+            if (StartInRelto() and self.EndgameSolved()):
```

Comment:
```suggestion
            if (self.EndgameSolved()):
```

This is a puzzle design change that I think @Deledrius is taking issue with. Let's split this out into a separate PR so the bug fix portion can be accepted immediately.

Reply:
Ive reverted the change,